src/main/scala/com/chaordicsystems/cactus/Parser.scala

```diff
@@ -0,0 +1,60 @@
+package com.chaordicsystems.cactus
+
+import com.chaordicsystems.cactus.Operator._
+import com.chaordicsystems.cactus.Validator._
+import com.sksamuel.elastic4s.ElasticDsl.{bool, must, should}
+import com.sksamuel.elastic4s._
+import org.json4s._
+import org.json4s.jackson.JsonMethods._
+
+object Parser {
+  implicit val formats = DefaultFormats
+
+  def AND(args: List[QueryDefinition]): QueryDefinition = bool { must(args) }
+
+  def OR(args: List[QueryDefinition]): QueryDefinition = bool { should(args) minimumShouldMatch 1 }
+
+  def handleUnary(operation: JValue, typeEnabled: Boolean): QueryDefinition = {
+    val (op, field, args) = validateUnaryOperation(operation)
+    op match {
+      case Operator.NE => field NE (args, typeEnabled)
+      case Operator.EQ => field EQ (args, typeEnabled)
+      case Operator.LT => field LT (args, typeEnabled)
+      case Operator.GT => field GT (args, typeEnabled)
+      case Operator.LE => field LE (args, typeEnabled)
+      case Operator.GE => field GE (args, typeEnabled)
```

Comment:
It was already checked before. First it checks for Unary Operators, then it send to this method, so it'll always match.